Resources/Prototypes/Entities/Clothing/Uniforms/jumpsuits.yml

```diff
@@ -1093,6 +1093,17 @@
   - type: Clothing
     sprite: Clothing/Uniforms/Jumpsuit/musician.rsi
 
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformJumpsuitERTChaplain
+  name: ERT chaplain uniform
+  description: A special suit made for the elite chaplain under CentCom.
```

Comment:
The grammar here is off. It would be better to write "A special suit made for the elite religious forces that work under CentCom", or something similar.

Reply:
Hows "A special suit made for Central Command's elite chaplain corps."